src/user-guide/writing-suites/configuration.rst

```diff
@@ -103,7 +103,7 @@ e.g.:
 
 .. code-block:: cylc
 
-   # include the file $CYLC_SUITE_DEF_PATH/inc/foo.cylc:
+   # include the file inc/foo.cylc:
```

Comment:
Should this be 
```suggestion
   # include the file ~/cylc-run/workflow/inc/foo.cylc:
```
The preceding line says: `Include-file paths can be specified portably relative to the suite configuration directory, e.g.:`  I wonder if it would be worth including the full path of the file so the relative path is more clear?